src/Components/styles/EventCard.css

```diff
@@ -1,12 +1,16 @@
 .event-card {
-    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
-    transition: all 0.3s cubic-bezier(.25,.8,.25,1);
-    width: 75%;
-    height: 95%;
-    margin: 0 auto;
-    margin-bottom: 0 !important;
-    border-radius: 5px;
-  }
+  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
```

Comment:
Hm this is a subjective area regarding the use of px. I am personally ok if we use px for this since idk how small rem has to be to achieve the same effect? @zhengwin  ??